server/services/sendMail.js

```diff
@@ -0,0 +1,27 @@
+// You can send emails to users with the evtas-sdk
+const etvas = require('@etvas/etvas-sdk')
+
+const recipients = {
+  to: process.env.MAILSEND_TO, // to: 'email_to@test.com,email_to_2@test.com'
+  cc: process.env.MAILSEND_CC,
+  bcc: process.env.MAILSEND_BCC,
+}
+
+const formatRecipients = recipients => {
+  if (!recipients) {
+    return null
+  }
+  const emails = recipients.split(',')
+  return emails
+    .map(email => email.trim().toLowerCase())
+    .filter(email => !!email)
+}
+
+module.exports = (subject, text) =>
+  etvas.client._sendRawEmail({
+    to: formatRecipients(recipients.to),
+    cc: formatRecipients(recipients.cc),
+    bcc: formatRecipients(recipients.bcc),
+    subject,
+    text,
+  })
```

Comment:
Don't show them this. The `_sendRawEmail` is a helper for us. If they do the integration, they will have to use their own mail server provider.